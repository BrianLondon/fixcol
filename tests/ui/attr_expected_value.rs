use fixcol_derive::ReadFixed;

#[derive(ReadFixed)]
struct Thing {
    #[fixcol(width = 5, align =, default = "none")]
    field: String,
}

pub fn main() {}
