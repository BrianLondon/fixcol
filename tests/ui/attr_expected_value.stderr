error: fixcol-derive error: Expected identifier or literal.
 --> tests/ui/attr_expected_value.rs:5:32
  |
5 |     #[fixcol(width = 5, align =, default = "none")]
  |                                ^
